core/Test/Tasty/Options/Core.hs

```diff
@@ -93,6 +94,15 @@ mkTimeout n =
   Timeout n $
     showFixed True (fromInteger n / (10^6) :: Micro) ++ "s"
 
+newtype HideProgress = HideProgress { getHideProgress :: Bool }
```

Comment:
Please add `@since 1.5` and haddock description. How does this option interact with `AnsiTricks`?